packages/framework/src/Component/Image/ImageFactory.php

```diff
@@ -81,8 +86,8 @@ public function createMultiple(
         }
 
         $images = [];
-        foreach ($temporaryFilenames as $temporaryFilename) {
-            $images[] = $this->create($imageEntityConfig->getEntityName(), $entityId, $type, $temporaryFilename);
+        foreach ($temporaryFilenames as $key => $temporaryFilename) {
```

Comment:
$key is a generic name. What is actually in it? I'm guessing something like image id.

Reply:
Well it is array key generated by PHP